packages/core/src/utils/__tests__/did.test.ts

```diff
@@ -149,4 +177,16 @@ describe('Utils | Did', () => {
       expect(isDidIdentifier(didIdentifier)).toBe(false)
     })
   })
+
+  describe('getIndyDidFromVerficationMethod()', () => {
+    test.each(verificationMethod)('returns indy did from verification method', (method) => {
+      expect(getIndyDidFromVerficationMethod(method)).toBe(indyDid)
+    })
```

Comment:
with only one test it doesn't make sense to use `test.each`:

```suggestion
    test('returns indy did from verification method', () => {
      expect(getIndyDidFromVerficationMethod(verificationMethod)).toBe(indyDid)
    })
```